<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>785</width>
    <height>605</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>785</width>
    <height>605</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>汉仪文黑-85W</family>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>591</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>370</width>
     <height>540</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>370</width>
     <height>600</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>输入</string>
    </attribute>
    <widget class="QLineEdit" name="le_cmt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_savefile">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>保存到文件</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_pos">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>理之冠</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>时之沙</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>空之杯</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>生之花</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>死之羽</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pb_openfile">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>读取文件</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>主词条</string>
     </property>
     <widget class="QComboBox" name="cb_main">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="digit_main">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>70</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>321</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>副词条</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>1.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>4.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>2.</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_sub_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_sub_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_sub_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_sub_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="digit_sub_1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="digit_sub_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="digit_sub_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="digit_sub_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>5000.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="cb_aeffect">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>套装</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>注释</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_openimg">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>读取图片</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>540</height>
     </size>
    </property>
    <attribute name="title">
     <string>说明</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>361</width>
       <height>500</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>510</height>
      </size>
     </property>
     <property name="plainText">
      <string>读取data/compare/当前人物目录下head/glass/cup/flower/feather下文件进行穷举，需要保证每个文件夹至少有一个文件
比较结果显示文件名前缀

录入模式激活时，将不再从 圣遗物主词条及其他增益副词条、套装效果中读取信息

其他增益其他加成仍然有效

套装信息会累加并自动加载已定义的套装效果</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
